Sources/Amplitude/Utilities/HttpClient.swift

```diff
@@ -83,6 +86,9 @@ class HttpClient {
         requestPayload += "}"
         return requestPayload.data(using: .utf8)
     }
+    func getDate() -> Date {
```

Comment:
nit: missing newline

```suggestion

    func getDate() -> Date {
```

Reply:
I got `Trailing Whitespace Violation: Lines should not have trailing whitespace (trailing_whitespace)` by swiftlint if adding newline here